### PR TITLE
Updating simulation w/o custom parser

### PR DESCRIPTION
**Pull request by Jgmedina95**

### CODE REVIEW

File: mdagent/tools/base_tools/simulation_tools/create_simulation.py [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+from langchain import LLMChain, PromptTemplate
+from langchain.base_language import BaseLanguageModel
+from langchain.tools import BaseTool
+from pydantic import BaseModel, Field
+
+from mdagent.utils import PathRegistry
+
+
+class ModifyScriptUtils:
+    llm = langchain.chat_models.ChatOpenAI(
```
Comment (SamCox822) [inline mdagent/tools/base_tools/simulation_tools/create_simulation.py:14]: @Jgmedina95 can we make the llm either inherit the model type from tools llm or be an input?

File: mdagent/tools/base_tools/simulation_tools/create_simulation.py [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+
+class ModifyScriptInput(BaseModel):
+    query: str = Field(..., description="Simmulation required by the user")
+    script: str = Field(..., description=" path to the base script file")
+
+
+class ModifyScriptTool(BaseTool):
+    name: str = "ModifyScriptTool"
+    description: str = """This tool takes a base script and a user
```
Comment (SamCox822) [inline mdagent/tools/base_tools/simulation_tools/create_simulation.py:137]: @Jgmedina95 can you make this a bit more descriptive? I assume this is only for MD simulations, so it should be a bit more specific - might cause problems later

File: mdagent/tools/base_tools/preprocess_tools/clean_tools.py [hunk 3]
```diff
@@ -220,3 +221,157 @@ def _run(self, query: str) -> str:
...
+    args_schema: Type[BaseModel] = CleaningToolFunctionInput
+
+    path_registry: Optional[PathRegistry]
+
+    def _run(self, **input_args) -> str:
+        """Use the tool with specified operations."""
+        try:
+            ### No idea why the input is a dictionary with the key "input_args"
+            # instead of the arguments themselves
+            print(input_args)
```
Comment (SamCox822) [inline mdagent/tools/base_tools/preprocess_tools/clean_tools.py:276]: also can we remove any unnecessary print statements that were used for debugging?

**Review by SamCox822 — CHANGES_REQUESTED** [review]

File: mdagent/tools/base_tools/simulation_tools/create_simulation.py (hunk 2 above)
Comment (Jgmedina95) [reply mdagent/tools/base_tools/simulation_tools/create_simulation.py:137]: You mean the tool description itself?

File: mdagent/tools/base_tools/preprocess_tools/clean_tools.py (hunk 3 above)
Comment (Jgmedina95) [reply mdagent/tools/base_tools/preprocess_tools/clean_tools.py:276]: fair!

File: mdagent/tools/base_tools/simulation_tools/create_simulation.py (hunk 1 above)
Comment (Jgmedina95) [reply mdagent/tools/base_tools/simulation_tools/create_simulation.py]: done!

File: mdagent/tools/base_tools/simulation_tools/create_simulation.py (hunk 2 above)
Comment (Jgmedina95) [reply mdagent/tools/base_tools/simulation_tools/create_simulation.py]: done!

**Review by SamCox822 — APPROVED** [review]
looks good to me!

**Pull request merged**